DirectEmployers/MyJobs
# [PD 2761] setup ui to make initial api call for primary dimension

This task was for consuming the API given to me by Darrin for setting the initial dimensions for the starting points. The dimensions are loaded in the sidebar coming from the API. Also inside of here is the task for changing and updating the initial starting point data date-range which should be the last 30 days. That can be seen inside of the analytics/components/analyticsapp.jsx component. 
Ignore console logs and commented out code that will be displayed inside of ESLint.

To Test:

You may need to possibly run a dkm migrate.
Will also need to make sure that you run an NPM install as some of the package.json has changed.
When pulling up the analytics app in the BETA tab, you should see that the initial data is still coming from the API provided by Jonathan with a date range of the last 30 days BUT the sidebar is being populated by the starting point API provided by Darrin.
